inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -71,9 +71,20 @@ const int gna_header_magic = is_little_endian() ?  0x4d414e47 : 0x474e414d;
 
 ModelHeader GNAModelSerial::ReadHeader(std::istream &is) {
     is.exceptions(std::istream::failbit);
+    is.seekg(0, is.end);
+    auto stream_len = is.tellg();
+    if (stream_len == -1) {
+        THROW_GNA_EXCEPTION << "Can't open file to import";
```

Comment:
file name also can be traced out.

Reply:
We use `istream` not `fstream` so it may be not a file